src/rinterface.c

```diff
@@ -7565,21 +7566,23 @@ SEXP R_igraph_famous(SEXP name) {
   return result;
 }
 
-SEXP R_igraph_get_adjlist(SEXP graph, SEXP pmode) {
+SEXP R_igraph_get_adjlist(SEXP graph, SEXP pmode, SEXP ploops, SEXP pmultiple) {
 
   igraph_t g;
   igraph_integer_t mode=(igraph_integer_t) REAL(pmode)[0];
   SEXP result;
   int i;
   long int no_of_nodes;
   igraph_vector_t neis;
+  igraph_integer_t loops=(igraph_integer_t) REAL(ploops)[0];
```

Comment:
What is the reasoning for having an intermediary of type `igraph_integer_t` instead of casting directly to `igraph_loops_t` / `igraph_multiple_t`?

Reply:
No particular reason, it's a pattern already used in this function. It seems that this code could also be autogenerated, we can handle improvements then.